File: content/78/a.go
package main
// example 1
func f(s string) int {
   return len(s)
}
// example 2
var f2 = func(s string) int {
   return len(s)
}
// print
func main() {
   n := f("May")
   n2 := f2("May")
   println(n == 3, n2 == 3)
}

File: content/78/a.php
<?php
# example 1
function f(string $s): int {
   return strlen($s);
}
# example 2
$f2 = function (string $s): int {
   return strlen($s);
};
# example 3
$f3 = fn (string $s): int => strlen($s);
# print
$n = f('May');
$n2 = $f2('May');
$n3 = $f3('May');
var_dump($n == 3, $n2 == 3, $n3 == 3);
